src/Ether.Vsts/Types/VstsClientFactory.cs

```diff
@@ -23,47 +23,62 @@ public VstsClientFactory(IMediator mediator)
 
         public IVstsClient Client { get; private set; }
 
-        public async Task<IVstsClient> GetClient(string token = null)
+        public async Task<IVstsClient> GetClient(Guid organizationId, string token = null)
```

Comment:
Project should be able to override identity specified in the org.

Reply:
This will be implemented later in #55 as discussed offline.